docs/en/query_language/select.md

```diff
@@ -448,22 +448,42 @@ The table names can be specified instead of `<left_subquery>` and `<right_subque
 
 See the standard [SQL JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) description.
 
-**ANY or ALL strictness**
+#### Multiple JOIN
+
+ClickHouse supports multiple `JOIN` operations in one query. Creating a query, you can use the following types of syntax:
```

Comment:
Probably we need some words here how does ClickHouse make these joins. It's not clear for user why we create this part of doc. He expects that there's no difference in joining 2 or 3 or 10 tables. That is not truth. We should say ClickHouse rewrites multiple joins into combination of 2-table joins and make them straitforward. If you are joining 4 tables it will join 1st and 2nd then join result with 3th and at the last step it will join the 4th one.

The other important thing is how does filters from WHERE, PREWHERE, GROUP and ORDER BY, etc. are applied. ClickHouse tries to push down WHERE clauses into rewritten joins if it can. But it covers not much cases now. If it's not cover some case multiple join leads to huge memory usage and bad timings of the query execution cause it joins the first tables without filters and apply them only after the last join.